### PR TITLE
Feature/settgast/install all headers

### PR DESCRIPTION
**Pull request by rrsettgast**
The CHAI installation did not contain the CHAI headers. 

`install(DIRECTORY ${PROJECT_BINARY_DIR}/include DESTINATION . OPTIONAL)`

was added to copy the entire include directory from the build directory to the install directory

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -110,6 +108,7 @@ install(
 
 install(EXPORT chai-targets DESTINATION share/chai/cmake/)
 
+
```
Comment (davidbeckingsale) [inline src/CMakeLists.txt]: What's this for?

File: src/CMakeLists.txt [hunk 2]
```diff
@@ -97,9 +97,7 @@ install(
   FILES ${PROJECT_BINARY_DIR}/share/chai/cmake/chai-config.cmake
   DESTINATION share/chai/cmake/)
 
-install(
-  FILES ${PROJECT_BINARY_DIR}/include/chai/config.hpp
-  DESTINATION include/chai)
+install(DIRECTORY ${PROJECT_BINARY_DIR}/include DESTINATION . OPTIONAL)
```
Comment (davidbeckingsale) [inline src/CMakeLists.txt]: I don't think we want this to be optional.

Comment (rrsettgast) [reply src/CMakeLists.txt]: True. I’ll fix this.

File: src/CMakeLists.txt (hunk 1 above)
Comment (rrsettgast) [reply src/CMakeLists.txt]: Errr.  Uhhhh. Just decided to insert a random line break??  I’ll remove it.

**Comment by davidbeckingsale** [comment]
Fixed by #11

**Pull request closed without merging**